src/main/scala/io/viash/helpers/DependencyResolver.scala

```diff
@@ -144,14 +144,20 @@ object DependencyResolver {
     config.map{ c =>
       val path = c.info.get.config
       // fill in the location of the executable where it will be located
-      val executable = Paths.get(ViashNamespace.targetOutputPath("", platformId.get, c.functionality.namespace, c.functionality.name), c.functionality.name).toString()
+      val executable = platformId match {
+        case Some("nextflow") => Paths.get(ViashNamespace.targetOutputPath("", platformId.get, c.functionality.namespace, c.functionality.name), "main.nf").toString()
+        case _ => Paths.get(ViashNamespace.targetOutputPath("", platformId.get, c.functionality.namespace, c.functionality.name), c.functionality.name).toString()
+      }
```

Comment:
Similar to your previous code,
```suggestion
      val executableName = platformId match {
        case Some("nextflow") => "main.nf"
        case _ => c.functionality.name
      }
      val executable = Paths.get(ViashNamespace.targetOutputPath("", platformId.get, c.functionality.namespace, c.functionality.name), executableName).toString()
```